src/components/v5/common/ActionSidebar/partials/forms/EditColonyDetailsForm/hooks.ts

```diff
@@ -23,26 +30,63 @@ export const useEditColonyDetails = (
     name: DECISION_METHOD_FIELD_NAME,
   });
 
+  const defaultValues = useMemo<DeepPartial<EditColonyDetailsFormValues>>(
+    () => ({
+      colonyName: metadata?.displayName,
+      avatar: {
+        image: metadata?.avatar,
+        thumbnail: metadata?.thumbnail,
+      },
+      colonyDescription: metadata?.description || '',
+      createdIn: Id.RootDomain.toString(),
+      externalLinks: metadata?.externalLinks ?? [],
+    }),
+    [
+      metadata?.avatar,
+      metadata?.description,
+      metadata?.displayName,
+      metadata?.externalLinks,
+      metadata?.thumbnail,
+    ],
+  );
+
+  const validationSchema = useMemo(
```

Comment:
Could we maybe pull this into a function called `getEditColonyDetailsValidationSchema` instead of inlining the schema just because of defaultValues?